admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/mainContainer/component/AdminSideBarTabs.kt

```diff
@@ -0,0 +1,110 @@
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DUKAN_REQUEST,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DUKAN_REQUEST) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.DEPOSIT.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DEPOSIT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DEPOSIT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DEPOSIT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DEPOSIT) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.USERS_MANAGEMENT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT) }
+        )
+    }
+}
+
+@Composable
+private fun AdminSidebarItem(
+    title: String,
+    selectedIcon: Painter,
+    notSelectedIcon: Painter,
+    isSelected: Boolean,
+    onClick: () -> Unit
+) {
+    val animatedTitleColor by animateColorAsState(
+        targetValue =
+            if (isSelected)
+                Theme.colorScheme.shadeSecondary
+            else
+                Theme.colorScheme.brand.brand
+    )
+    Column(
+        modifier = Modifier
+            .padding(horizontal = 8.dp)
+            .fillMaxWidth()
+            .clickable { onClick() },
+        verticalArrangement = Arrangement.spacedBy(8.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Crossfade(targetState = isSelected)
+        { selected ->
+            Icon(
+                painter = if (selected) selectedIcon else notSelectedIcon,
```

Comment:
if condition can be removed outside  Icon as well
```suggestion
        Crossfade(targetState = isSelected) { selected ->
            val icon = if (selected) selectedIcon else notSelectedIcon
            Icon(
                painter = icon,
```